packages/admin-next/dashboard/src/v2-routes/products/product-detail/components/product-sales-channel-section/product-sales-channel-section.tsx

```diff
@@ -1,19 +1,19 @@
 import { Channels, PencilSquare } from "@medusajs/icons"
 import { Product } from "@medusajs/medusa"
 import { Container, Heading, Text, Tooltip } from "@medusajs/ui"
-// import { useAdminSalesChannels } from "medusa-react"
 import { Trans, useTranslation } from "react-i18next"
 import { ActionMenu } from "../../../../../components/common/action-menu"
+import { useSalesChannels } from "../../../../../hooks/api/sales-channels"
 
 type ProductSalesChannelSectionProps = {
   product: Product
 }
 
+// TODO: The fetched sales channel doesn't contain all necessary info
```

Comment:
What are you missing?

Reply:
@olivermrbl it only has the ID, but we do `*sales_channels` in the selection, I need to see why.